src/Resources/skeleton/registration/twig_email.tpl.php

```diff
@@ -0,0 +1,11 @@
+<h1>Thank you for registering!</h1>
```

Comment:
I'm thinking this should be less of a "registration confirmation" and more of a pure "email confirmation" email.

```suggestion
<h1>Hi! Please confirm your email!</h1>
```